src/schema/v2/me/triggerCampaignMutation.ts

```diff
@@ -19,7 +19,7 @@ const CAMPAIGN_IDS = {
   // New campaign IDs can be added by environment variable here
```

Comment:
Wondering if this should come from ENV (like the comment suggests), as `BRAZE_ART_QUIZ_CAMPAIGN_ID` (which can be set differently in dev/staging and prod) vs hard-coded.

Reply:
yep yep, lots of discussions have been happening about this and I'm planning to move these out into a s3 bucket as part of the future work around this `triggerCampaign` endpoint. We're launching the art quiz today and need emails to go out so it will have to wait unfortunately :sweat_smile: